src/app/core/filter/filter.service.ts

```diff
@@ -37,6 +54,7 @@ export class FilterService {
    * ```
    * @param filter a valid filter object, e.g. as provided by the `FilterComponent`
    */
+  // todo: check usage for array usage (typing not working)
```

Comment:
Can you explain this a bit more or create an issue for this?

Reply:
Deleted, not a todo anymore. -